CDP4IME/Shell.xaml

```diff
@@ -54,11 +54,6 @@
 
             <dxr:RibbonControl.PageHeaderItems>
 
-                <dxb:BarButtonItem Name="bAbout"
```

Comment:
Please revert this change. It does not do much for now, but hardly belongs in this PR.

Reply:
Done